ascend-sdk/src/main/java/ai/evolv/ascend/android/Allocator.java

```diff
@@ -2,26 +2,62 @@
 
 import android.support.annotation.NonNull;
 
-import ai.evolv.ascend.android.exceptions.AscendAllocationException;
-import com.google.common.util.concurrent.ListenableFuture;
+import com.google.common.util.concurrent.SettableFuture;
 import com.google.gson.JsonArray;
 import com.google.gson.JsonParser;
 
+import java.io.IOException;
+import java.util.concurrent.Future;
 import java.util.concurrent.TimeUnit;
 
+import ai.evolv.ascend.android.exceptions.AscendAllocationException;
+import okhttp3.Call;
+import okhttp3.Callback;
 import okhttp3.HttpUrl;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
+import timber.log.Timber;
```

Comment:
Avoid direct dependency on platform specific libraries.

Reply:
Ill remove Timber and adopt SLF4J for this PR, and then implement pluggable http client in the next pr.